### DISCUSSION TITLE
Friendly URLs Not Working

### DISCUSSION THREAD

**Discussion by im-L0r3** [e107inc/e107#4764] [General]
Under Site URLs if I set "Custom Pages" to any of the options other than "Default" which is "https://fqdn/page.php?1", then hit "Rebuild" results in pages when selected in a menu or URL typed manually fail with "The requested URL was not found on this server."

Im assuming the "Friendly" of "https://fqdn/page/page-title" should be creating the URL from the SEF URL?

I can imagine I have something misconfigured somewhere. My Google-foo has not been able to find a resolution.

**Comment by CaMer0n** [comment]
@im-L0r3 Is it all of the SEF URLs that are failing or just the "Custom pages" ones?

**Reply by im-L0r3** [reply]
All of them.

Pages load using the default, for example this works: https://l0r3.dev/page.php?id=29
The SEF URL set for that page is "Build-Paladin-Smiter"

If I set Site URLs > Profiles > Custom Pages, none of the alternate types work.
I believe in theory I should be able to set:
Site URLs > Profiles > Custom Pages > Friendly: https://l0r3.dev/page/page-title

Then I hit save and rebuild but none of the following work:
https://l0r3.dev/Build-Paladin-Smiter
https://l0r3.dev/page/Build-Paladin-Smiter
https://l0r3.dev/Build-Paladin-Smiter.php

I must be missing a setting or configuration change needed somewhere...

**Comment by CaMer0n** [comment]
@im-L0r3 Thanks for the info. If you add the following code on a new line of e107_config.php , does it help?

    define('e_MOD_REWRITE',true);

Also, are you sure mod_rewrite is installed on your server?

**Reply by im-L0r3** [reply]
Thanks to your hint, it gave me a direction to dig into...

mod_rewrite was already enabled

The solution that solved it was in my sites vhost configuration file.

I changed:
```
        <Directory "/var/www/html/l0r3_dev">
                Options Indexes
        </Directory>
```

To:
```
        <Directory "/var/www/html/l0r3_dev">
                Options Indexes FollowSymLinks MultiViews
                AllowOverride All
                Require all granted
        </Directory>
```

THANK YOU! Im an Escalation Engineer at a lager cloud provider, and I will definitely recommend e107 to my customers looking for a CMS.

**Reply by CaMer0n** [reply]
Thank you @im-L0r3 . I'm happy you found the issue. :+1:
